src/components/professional/ProjectClose.vue

```diff
@@ -42,7 +42,7 @@ export default defineComponent({
       }
     },
     async ProjectStatus() {
-      const response = await fetch('http://localhost:5235/api/project-status', {
+      const response = await fetch('http://localhost:5235/api/project-close_fail', {
```

Comment:
Nincsen ilyen API call.

Reply:
Ment hozza a backend is abba mar van, elobb nem kuldte fel